bump.sh

```diff
@@ -55,8 +55,21 @@ if [[ $version == "patch" ]]; then
   sed -i "s|$version_header|\1|" ./CHANGELOG.md
 fi
 
-# Add a section to the changelog that state the version of CDK being used
 version_header="# \[$new_version](.*) (.*)"
+
+# Add a section to the changelog that states the supported Deadline versions
+DEADLINE_RELEASE_NOTE_URL="https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/release-notes.html"
+MIN_DEADLINE_VERSION=$(grep 'MINIMUM_SUPPORTED_DEADLINE_VERSION_STRING =' $(find ./packages/aws-rfdk/lib/ -name '*.ts') | tr ' ' '\n' | grep "'" | tr -d "';")
+MAX_DEADLINE_VERSION=$(node ./scripts/getLatestDeadlineVersion.ts)
```

Comment:
Could we do something similar where we run a node script to obtain the minimum supported version instead of grepping?

Reply:
Sure, I'll switch things up.